modules/collecting-gitops-debugging-data.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assembly:
+//
+// * cicd/gitops/collecting-debugging-data-for-support.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-customize-argo-cd-consolelink_{context}"]
+= Collecting debugging data for {gitops-title}
+
+Use the `oc adm must-gather` CLI command to collect the following details about the cluster that is associated with {gitops-title}:
+
+* The subscription and namespace of the {gitops-title} Operator
+* The namespaces where ArgoCD objects are available and the objects in those namespaces, such as `ArgoCD`, `Applications`, `ApplicationSets`, `AppProjects`, and `configmaps`
+* A list of the namespaces that are managed by {gitops-title} Operator and resources from those namespaces
+* All GitOps-related custom resource objects and definitions
+* Operator and Argo CD logs
+* Warning and error-level events
+
+.Prerequisites
+* You have logged in to the {product-title} cluster as an administrator.
+* You have installed the {product-title} CLI (`oc`).
```

Comment:
@reginapizza Should we also add 'GitOps operator is installed on the cluster'  as a prerequisite?

Reply:
Incorporated this suggestion @varshab1210